eva_submission/eload_ingestion.py

```diff
@@ -74,14 +74,16 @@ def ingest(
         if do_accession:
             self.eload_cfg.set(self.config_section, 'accession', 'instance_id', value=instance_id)
             self.update_config_with_hold_date(self.project_accession)
-            self.run_accession_workflow(vcf_files_to_ingest)
+            output_dir = self.run_accession_workflow(vcf_files_to_ingest)
+            shutil.rmtree(output_dir)
```

Comment:
**Judgment call:** I am guessing this is being done here because this is the last step in the automation (for now). But if we tack on another step to the automation, will this piece of code be harmful? While it is a good strategy to trim down Nextflow directories from time to time, I would vote for doing it **outside** the automation scripts (ex: cronjob that combs through ELOADs that have been loaded and culls out old Nextflow directories).

Reply:
It's being done after each Nextflow workflow, so after the accessioning here and after the load step below, each of which has its own work directory.  This is consistent with elsewhere in the submission automation (e.g. [in validation](https://github.com/EBIvariation/eva-submission/blob/master/eva_submission/eload_validation.py#L42)).  Any additional steps would be either added to the individual workflows, done in Python outside of Nextflow, or done as an additional workflow which would have its own work directory.

Reply:
:+1:

Reply:
Since the topic came up, I'm going to chime in as well.
Overall we need to be (and are most time) very aggressive when it come to delete Nextflow work directories. They can get very big and are almost never useful later. So I would actually be in favour of *additional* method to clean them up.
The only exception that we need to care for (in a separate ticket of course) is our ability to resume process when they failed.
We also need to make sure that all files that are actually needed (logs, properties, output) are copied out to their final location.